x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -24,10 +24,18 @@ export const LogEntryFieldColumn: React.FunctionComponent<LogEntryFieldColumnPro
   isWrapped,
 }) => {
   const value = useMemo(() => JSON.parse(encodedValue), [encodedValue]);
-
+  const formattedValue = Array.isArray(value) ? (
+    <ul>
```

Comment:
<img width="257" alt="Screen Shot 2019-06-12 at 11 34 15 AM" src="https://user-images.githubusercontent.com/1445834/59365144-0b467d80-8d06-11e9-8d50-97dd82b4b8bf.png">

Here they are comma-separated with spaces in between. Does that work? I'm not sure how I feel about having the addresses break in the middle to wrap onto the next line, but it does save some vertical space.

Reply:
<img width="1449" alt="Screen Shot 2019-06-12 at 12 11 07 PM" src="https://user-images.githubusercontent.com/1445834/59368000-41d2c700-8d0b-11e9-92de-18a3a6de4a35.png">

Tweaked slightly, using `1ex` whitespace instead of just putting a space character

Reply:
I would imagine that the wrapping within the address occurs because the `:` is considered a word delimiter. Agreed that it's not ideal, but not unreadable either. At some point we might want to have semantic formatting for value, e.g. to prevent wrapping of ip addresses or to link geo coords to maps and so on. I think this is fine for now though, thank you!